Recrutify/Recrutify.Services/Validators/ProjectResultValidator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Linq;
+using FluentValidation;
+using Recrutify.DataAccess.Models;
+
+namespace Recrutify.Services.Validators
+{
+    public class ProjectResultValidator : AbstractValidator<ProjectResult>
+    {
+        public ProjectResultValidator()
+        {
+            RuleFor(f => f.Feedbacks.FirstOrDefault())
+                .NotNull()
+                .Must(BeAValidData)
+                .WithMessage("Time for editing is over");
+            RuleFor(f => f.Status)
+                .IsInEnum()
+                .Must(s => !new[] { Status.Accepted, Status.Declined, Status.WaitingList, }.Contains(s))
+                .WithMessage("Status is not available for update");
```

Comment:
Feedback is not available for update for candidate in current status

Reply:
done